Clippit/PowerPoint/PresentationBuilderTools.cs

```diff
@@ -331,8 +331,6 @@ internal static void AddRelationships(OpenXmlPart oldPart, OpenXmlPart newPart,
                     var newRid = Relationships.GetNewRelationshipId();
                     var oldHyperlink = oldPart.HyperlinkRelationships.FirstOrDefault(h => h.Id == relId);
                     if (oldHyperlink is null) {
-                        //TODO Issue with reference to another part: var temp = oldPart.GetPartById(relId);
-                        RemoveContent(newContent, e.Name, relId);
```

Comment:
This case is interesting... We copy hyperlinks relationships and cannot find hyperlinks inside the source document by it is `relId`. 

The code assumes that the link is broken and removes the relationship from the target file. Can you create new slide manually from scratch that triggers this if statement during slide publishing?